simqueue/models.py

```diff
@@ -57,6 +59,7 @@ def as_json(self):
             'timestamp_completion': self.timestamp_completion,
             'provenance': self.provenance,
             'resource_usage': self.resource_usage,
+            'tags': [tag.name for tag in self.tags.get_query_set()],
```

Comment:
I think I am using a newer version of taggit than you, because I get an error
```python
AttributeError: '_TaggableManager' object has no attribute 'get_query_set'
```
In my version (0.22.1) it expects the spelling 'get_queryset'